docs/HeliumParamSpecs.md

```diff
@@ -0,0 +1,295 @@
+# Helium Parameter Specs Documentation
+
+## System Overview
+
+Helium is WebAPI reference application designed to "fork and code" in c#, java, typescript languages with the following features:
```

Comment:
I dont think we need the system overview since that's in the Readme.  Maybe repurpose this section or choose another section (not sure which one), to briefly describe the goals/intent of the document - defining valid/invalid parameters and defining the resulting behavior of the Helium API.

Reply:
Updated